R/R-Formula/Portfile

```
@@ -0,0 +1,16 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           R 1.0
+
+R.setup             cran cran Formula 1.2-4
+revision            0
+categories-append   math
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+license             {GPL-2 GPL-3}
```

Multiple licenses should not be enclosed in curly braces.

Curly braces in Tcl take multiple words and turn them into a single token.

```suggestion
license             GPL-2 GPL-3
```

Please make a follow-up PR correcting this for any ports that may have already been committed.